massdriver-application-azure-app-service/_variables.tf

```diff
@@ -0,0 +1,38 @@
+variable "acr" {
```

Comment:
It looks like we’re naming this specifically for the azure container registry and then using a data to look up below. Does that mean it’s not possible to run public docker hub example apps here?

Reply:
![image](https://user-images.githubusercontent.com/8037512/189706069-3f1b6213-30b5-499e-b4d0-2c6962316c9e.png)
It looks like it's possible within the UI, but I'm struggling to see how to do it with Terraform. Looking into it right now.

Reply:
@coryodaniel I have a public Dockerhub repo linked up using Terraform. It was ridiculously vague in the documentation for how to set it up: 

![image](https://user-images.githubusercontent.com/8037512/189716030-6cd3e75b-1bbb-4991-829e-b52c45b1df20.png)

Using my [personal public repo](https://hub.docker.com/repository/docker/mclacore/test-repo). Should I change up the variables a bit to be more flexible? I think I can set up an enum for the type of repo to host Docker files, and set up 2 dynamic blocks to support both DockerHub and ACR.